apps/include/pcl/apps/vfh_nn_classifier.h

```diff
@@ -38,231 +38,260 @@
 #pragma once
 
 #include <fstream>
-#include <pcl/point_types.h>
-#include <pcl/io/pcd_io.h>
-#include <pcl/features/vfh.h>
-#include <pcl/features/normal_3d.h>
-#include <pcl/apps/nn_classification.h>
 #include <pcl/PCLPointCloud2.h>
+#include <pcl/apps/nn_classification.h>
```

Comment:
Separate the pcl and stdlib headers (and pcl comes before since it's just one stdlib header)

Reply:
I will wait for the result of #3904